database/seeds/October16NewCategoryTableSeeder.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+use Carbon\Carbon;
```

Comment:
I already saw it in previous PR. Probably you got it because you committed these changes in master branch and created issue branch from master branch. Let me first merge that PR so that there are no conflicts.